<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AcceptandPayOfferListPage</class>
 <widget class="QDialog" name="AcceptandPayOfferListPage">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>611</width>
    <height>700</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>700</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Pay Offer</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="labelExplanation">
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QFormLayout" name="formLayout">
     <property name="fieldGrowthPolicy">
      <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
     </property>
     <property name="verticalSpacing">
      <number>1</number>
     </property>
     <item row="0" column="0">
      <widget class="QPushButton" name="imageButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>300</width>
         <height>300</height>
        </size>
       </property>
       <property name="baseSize">
        <size>
         <width>1</width>
         <height>0</height>
        </size>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="iconSize">
        <size>
         <width>300</width>
         <height>300</height>
        </size>
       </property>
       <property name="autoRepeat">
        <bool>false</bool>
       </property>
       <property name="autoDefault">
        <bool>true</bool>
       </property>
       <property name="default">
        <bool>false</bool>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <layout class="QFormLayout" name="formLayout_3">
       <property name="fieldGrowthPolicy">
        <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
       </property>
       <item row="1" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Offer ID:</string>
         </property>
         <property name="buddy">
          <cstring>offeridEdit</cstring>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="offeridEdit">
         <property name="toolTip">
          <string>The value associated with this offer.</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QPushButton" name="lookupButton">
         <property name="toolTip">
          <string>Lookup the OfferID from the blockchain DB</string>
         </property>
         <property name="text">
          <string>&amp;Lookup Offer</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Quantity:</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QLineEdit" name="qtyEdit"/>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="aliasLabel">
         <property name="text">
          <string>Alias:</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QComboBox" name="aliasEdit"/>
       </item>
       <item row="6" column="1">
        <widget class="QLabel" name="aliasDisclaimer">
         <property name="text">
          <string/>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="8" column="0">
        <widget class="QLabel" name="label_11">
         <property name="text">
          <string>Notes:</string>
         </property>
        </widget>
       </item>
       <item row="8" column="1">
        <widget class="QPlainTextEdit" name="notesEdit">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>90</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="7" column="1">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_3">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Offer Details</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <layout class="QVBoxLayout" name="verticalLayout_2"/>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>Merchant:</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLineEdit" name="sellerEdit">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label_14">
       <property name="text">
        <string>Merchant Rating:</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QLineEdit" name="sellerRatingEdit">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Title:</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QLineEdit" name="infoTitle">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="8" column="1">
      <layout class="QVBoxLayout" name="verticalLayout_3"/>
     </item>
     <item row="9" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Category:</string>
       </property>
      </widget>
     </item>
     <item row="9" column="1">
      <widget class="QLineEdit" name="infoCategory">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="10" column="1">
      <layout class="QVBoxLayout" name="verticalLayout_4"/>
     </item>
     <item row="11" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Price:</string>
       </property>
      </widget>
     </item>
     <item row="11" column="1">
      <widget class="QLineEdit" name="infoPrice">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="12" column="1">
      <layout class="QVBoxLayout" name="verticalLayout_5"/>
     </item>
     <item row="13" column="0">
      <widget class="QLabel" name="label_12">
       <property name="text">
        <string>Currency:</string>
       </property>
      </widget>
     </item>
     <item row="13" column="1">
      <widget class="QLineEdit" name="infoCurrency">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="14" column="1">
      <layout class="QVBoxLayout" name="verticalLayout_6"/>
     </item>
     <item row="15" column="0">
      <widget class="QLabel" name="label_10">
       <property name="text">
        <string>Alias for Exchange Rate Peg:</string>
       </property>
      </widget>
     </item>
     <item row="15" column="1">
      <widget class="QLineEdit" name="aliasPegEdit">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="16" column="0">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Quantity Remaining:</string>
       </property>
      </widget>
     </item>
     <item row="16" column="1">
      <widget class="QLineEdit" name="infoQty">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="17" column="0">
      <widget class="QLabel" name="label_13">
       <property name="text">
        <string>Quantity Sold:</string>
       </property>
      </widget>
     </item>
     <item row="17" column="1">
      <layout class="QVBoxLayout" name="verticalLayout_7">
       <item>
        <widget class="QLineEdit" name="infoSold">
         <property name="enabled">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="18" column="0">
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>Description:</string>
       </property>
      </widget>
     </item>
     <item row="18" column="1">
      <widget class="QPlainTextEdit" name="infoDescription">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>700</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="19" column="1">
      <layout class="QVBoxLayout" name="verticalLayout_8"/>
     </item>
     <item row="20" column="1">
      <layout class="QVBoxLayout" name="verticalLayout_9"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="acceptButton">
     <property name="toolTip">
      <string>Purchase this offer</string>
     </property>
     <property name="text">
      <string>&amp;Accept Offer</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../syscoin.qrc"/>
 </resources>
 <connections/>
</ui>
